clients/src/main/java/org/apache/kafka/common/metrics/stats/Min.java

```diff
@@ -22,7 +22,7 @@
 public class Min extends SampledStat {
 
     public Min() {
-        super(Double.MIN_VALUE);
+        super(Double.MAX_VALUE);
```

Comment:
No worries. I noticed that too, and I think it would be more consistent to use INFINITY, but the convention seems to be that the `combine` implementation uses the same value for the de facto default (even though it doesn't make use of `initialValue`). Min() is already using Double.MAX_VALUE in `combine`, so also changing that felt like a bigger change (and possibly unwelcome if users were already handling Double.MAX_VALUE in their metrics tooling).